docs/conceptual/seq.tail['t]-function-[fsharp].md

```diff
@@ -0,0 +1,76 @@
+---
+title: Seq.tail<'T> Function (F#)
+description: Seq.tail<'T> Function (F#)
+keywords: visual f#, f#, functional programming
+author: liboz
+---
+
+# Seq.tail<'T> Function (F#)
+
+Returns the sequence without the first element.
```

Comment:
I'm not sure "the sequence" is crystal clear here.  Something more along the lines of "a new sequence composed of the input sequence without its first element" makes the behavior more clear.